plugins/codemirror/dialogs/codemirrordialog.js

```diff
@@ -5,6 +5,13 @@ CKEDITOR.dialog.add('codemirrordialog', function (editor) {
 	var scaley = 0.7;
 	var height = size.height * scaley;
 	var width = size.width * scalex;
+	var ALERT_REGEX = /alert\((.*?)\)/;
+	var INNER_HTML_REGEX = /innerHTML\s*=\s*.*?/;
+	var PHP_CODE_REGEX = /<\?[\s\S]*?\?>/g;
+	var ASP_CODE_REGEX = /<%[\s\S]*?%>/g;
+	var ASP_NET_CODE_REGEX = /(<asp:[^]+>[\s|\S]*?<\/asp:[^]+>)|(<asp:[^]+\/>)/gi;
+	var HTML_TAG_WITH_ON_ATTRIBUTE_REGEX = /<[^>]+?(\s+\bon\w+=(?:'[^']*'|"[^"]*"|[^'"\s>]+))*\s*\/?>/gi;
+	var ON_ATTRIBUTE_REGEX = /(\s+\bon\w+=(?:'[^']*'|"[^"]*"|[^'"\s>]+))/gi;
```

Comment:
Where did you get these from? Do we already do something similar in liferay-portal somewhere?

Reply:
Hello @bryceosterhaus 

I took this from here: https://liferay.atlassian.net/browse/LPD-2161.
This is done for Rich Text Fields, and I asked the team to see if there component could cover this particular issue, but they came back saying it couldn't. 
https://liferay.atlassian.net/browse/LPP-55225?focusedCommentId=2721890

Let me know if there are any further questions or comments about this.

Reply:
Okay good! Just wanted to make sure these regexes were at least being used in some other place so that we are testing so many new ones at once.